community-roadmap/README.md

```diff
@@ -0,0 +1,13 @@
+# Community Roadmap for Zendesk Help Center
+
+This project provides a public roadmap feature in Zendesk Help Center, where the product roadmap items are driven by community posts.
+
```

Comment:
It's up to you, but I think it may be more helpful to provide more context here:

- What problem does this solve? Why would a user be interested in doing this? Does it let them identify roadmap items based on community interest?
- At a high-level, how do community posts determine the roadmap items?
- Are there some example screenshots or a demo link we can share?
- What prerequisites or skills are needed to set up the feature? Do users need a specific plan? Do they need experience with React or node.js?

Reply:
Good feedback, thx. I've updated the READMEs, please take a look.